app/models/address.rb

```diff
@@ -0,0 +1,19 @@
+class Address < ApplicationRecord
+  VALID_FIELDS = /[a-zA-Z]/.freeze
+
+  validates :city, :country, :state, :street, presence: true, format: { with: VALID_FIELDS,
+                                                                        message: 'is invalid, must be 8 digits long' }
```

Comment:
is it valid error message? looks like all that column a strings, no digitnt

Reply:
done